src/compiler/checker.ts

```diff
@@ -8193,7 +8193,7 @@ namespace ts {
             const res = tryGetDeclaredTypeOfSymbol(symbol);
             if (res) {
                 return checkNoTypeArguments(node, symbol) ?
-                    res.flags & TypeFlags.TypeParameter ? getConstrainedTypeVariable(<TypeParameter>res, node) : res :
+                    res.flags & TypeFlags.TypeParameter ? getConstrainedTypeVariable(<TypeParameter>res, node) : getRegularTypeOfLiteralType(res) :

```

Comment:
Again, here I would just expect to find the regular type always.

Reply:
Again, as I said above, this makes more sense. Just like how string and number literal's always start fresh, the enum declaration itself is the literal for an enum. We have the ability to widen and strip freshness expressed well in most places in typespace for literals, we don't exactly have a good idea of places where we could unwiden-when-entering-an-expression-context, by comparison. Here, specifically, we needed to strip freshness anew because when only expression literals were fresh, typeof and a literal in a literal type was the only way they could enter typespace (and we strip there). Enum members, however, do not use typeof (they effectively direct straight to the expression type - both typespace and expression enum ref types resolve from the same underlying cache field), so they need the freshness stripped whenever we reference them in typespace, which must happen via a reference, so here.